release/version.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
This has been done already, why do this again a different way?
https://github.com/coreos/coreos-baremetal/blob/master/git-version
https://github.com/coreos/dex/blob/master/git-version